cmd/swarmctl/service/inspect.go

```diff
@@ -80,7 +80,7 @@ func printServiceSummary(service *api.Service, running int) {
 		}
 	}
 	if len(service.Spec.Task.Networks) > 0 {
-		fmt.Fprintf(w, "  Networks:\t")
+		fmt.Fprintln(w, "  Networks:\t")
```

Comment:
Hmm, it does look like a bug that they print a line ending in `\t`. Can you test it out and see if the output looks correct on one line? Otherwise the `\t` should be removed.